architecture/core_concepts/haproxy-router.adoc

```diff
@@ -0,0 +1,76 @@
+[[architecture-core-concepts-haproxy-router]]
+= HAProxy Router Plug-in
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+A router is one way to
+xref:../../dev_guide/getting_traffic_into_cluster.adoc#getting-traffic-into-cluster[get
+traffic into the cluster]. The HAProxy Template Router plug-in is one of the
+available
+xref:../../architecture/core_concepts/routes.adoc#available-router-plug-ins[router
+plugins].
+
+[[haproxy-template-router]]
+== HAProxy Template Router
+
+The template router has two components:
+
+* a wrapper that watches endpoints and routes and causes a HAProxy reload based on changes.
+* a controller that builds the HAProxy configuration file based on routes and endpoints.
+
+[NOTE]
+====
+The link:https://cbonte.github.io/haproxy-dconv/1.5/configuration.html[HAProxy router] uses version 1.5.18.
+====
+
+The controller and HAProxy are housed inside a pod, which is managed by a
+deployment configuration. The process of
+xref:../../install_config/router/index.adoc#[setting up the router] is automated
```

Comment:
We can get rid of the `#` if it is not related to a specific section on that page